style.css

```diff
@@ -616,23 +613,219 @@ textarea::placeholder {
 }
 
 .footer {
-  height: 1rem;
-  width: 100%;
-  padding-bottom: 0.5rem;
-  display: flex;
-  flex-direction: column;
-  justify-content: flex-end;
-  align-items: center;
-  background: #6070ff;
+  display: none;
 }
 
-.footer_bar {
-  width: 8rem;
-  height: 0.3rem;
-  background: #000;
+@media only screen and (min-width: 768px) {
+  .header {
+    padding-right: 9.625rem;
+  }
+
+  .name__int {
+    font-size: 3rem;
+    font-weight: 700;
+    line-height: 3.75rem;
+  }
+
+  .desc__int {
+    font-size: 1.25rem;
+    font-weight: 400;
+    line-height: 1.75rem;
+  }
+
+  .introduction {
+    margin-left: 12.5%;
+    width: 75%;
+    height: 300px;
+  }
+
+  .banner {
+    padding-top: 0.5rem;
+    padding-left: 1rem;
+  }
+
+  .mainfo {
+    margin-left: 1.875rem;
+    width: 45%;
+  }
+
+  .canopy {
+    flex: 0.35;
+  }
+
+  .fcbk {
+    flex: 0.45;
+  }
+
+  .uber {
+    flex: 0.25;
+  }
+
+  .bed,
+  .y2015 {
+    color: #6b7786;
+  }
+
+  .tonic {
+    margin-bottom: 0;
+  }
+
+  .grid__0 {
+    order: 0;
+  }
+
+  .grid__1 {
+    display: none;
+  }
+
+  .grid__2 {
+    display: none;
+  }
+
+  .grid__3 {
+    display: none;
+  }
+
+  .grid__4 {
+    display: none;
+  }
+
+  .grid__5 {
+    order: 1;
+  }
+
+  .grid__6 {
+    order: 2;
+  }
+
+  .grid__7 {
+    order: 3;
+  }
+
+  .flex__10 {
+    order: 1;
+  }
+
+  .flex__14 {
+    order: 1;
+  }
+
+  .knowledges {
+    margin-top: 4.5rem;
+    height: 450px;
+  }
+
+  .sleng,
+  .sframw,
+  .sskill {
+    margin-left: 1rem;
+  }
+
+  .zleng {
+    padding-bottom: 0;
+    padding-right: 2rem;
+  }
+
+  .zframw {
+    padding-top: 0;
+    padding-bottom: 0;
+    padding-right: 2rem;
+  }
+
+  .zskill {
+    padding-bottom: 0;
+    padding-right: 2rem;
+  }
+
+  .app {
+    padding-left: 0;
+  }
+
+  .contact_exp {
+    width: 45%;
+    margin-left: 27.5%;
+  }
+
+  .contactme form li input {
+    width: 35%;
+    margin-left: 32.5%;
+    margin-top: 20px;
+    border-radius: 0;
+    height: 48px;
+    padding: 0;
+  }
+
+  .contactme form li textarea {
+    width: 35%;
+    border-radius: 0;
+    height: 170px;
+  }
+
+  .textarea {
+    margin-left: 32.5%;
+  }
+
+  .contact::before {
+    content: "";
+    position: absolute;
+    background-color: #6070ff;
+    background-image: url(images/contact\ form\ background\ shapes.svg);
+    background-size: contain;
+    background-position: right;
+    background-origin: content-box;
+    background-repeat: no-repeat;
+    border-top-left-radius: 8rem;
+    top: 0;
+    left: 0;
+    z-index: -1;
+  }
+
+  .contact {
+    padding-top: 9rem;
+    height: 846px;
+  }
+
+  .contactme {
+    max-height: 846px;
+    margin-top: -256px;
+  }
+
+  .allaboutme {
+    height: 924px;
+    margin-left: 0;
+  }
```

Comment:
- [ ] Furthermore, the height of the section is so much greater than the one in the Figma design 😎 . Preferably though you can use the simplest method. I suggest removing the fixed height so that the contents of the section can easily blend into different screen sizes 👌 . This will help us match the Figma design and make the portfolio more professional ☺️ .

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/222515496-f9ae92b9-9c0e-484e-ab33-1ff0fe9bdcb2.png)

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/222515630-bdd989c1-33a0-47dc-9f34-c3c27d0b29b1.png)